Sources/TinkoffInvestSDK/SDK/TinkoffInvestSDK.swift

```diff
@@ -38,4 +38,6 @@ public class TinkoffInvestSDK {
     public lazy var stopOrdersService: StopOrdersService = GRPCStopOrdersService(tokenProvider: commonTokenProvider, appName: appName)
 
     public lazy var sandboxService: SandboxService = GRPCSandboxService(tokenProvider: sandboxTokenProvider, appName: appName)
```

Comment:
Also I've just noticed that sandboxTokenProvider now required for SDK initialization, how about we make it optional and pass nil as default value?